src/api/orderBookApi.ts

```diff
@@ -0,0 +1,33 @@
+export interface OrderBookApi {
+  fetchAll: () => Promise<any[]>;
+}
+
+export class RestOrderBookApi implements OrderBookApi {
+  fetchAll = () => Promise.resolve([]);
+}
+
+// tslint:disable-next-line:max-classes-per-file
+export class MockOrderBookApi implements OrderBookApi {
+  fetchAll = () => Promise.resolve<any[]>(generateOrders());
+}
+
+export default OrderBookApi;
+
+function generateOrders(): any[] {
```

Comment:
plz use arrow functions, it's preferrable

`const generateOrders = () => {...impl}`

Plus could we have this as a static method in MockApi or extract it to some utils?